tools/resourcedocsgen/cmd/metadata.go

```diff
@@ -369,8 +370,8 @@ func isComponent(keywords []string) bool {
 	return getTagFromKeywords(keywords, "kind/component") != nil
 }
 
-func isFeaturedPackage(package string) bool {
-	return slices.Contains(featuredPackages, package)
+func isFeaturedPackage(pkgName string) bool {
+	return git stslices.Contains(featuredPackages, pkgName)
```

Comment:
Looks like something has gone wrong here...

It seems like there is an extra `git st` in there.

Reply:
People like me need pre-commit hooks. 🤦